core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -844,7 +847,7 @@ public PersistenceState getPersistenceState(long fileId) throws FileDoesNotExist
       return ret;
     } finally {
       // finally runs after resources are closed (unlocked).
-      waitForJournalFlush(flushCounter);
+      waitForJournalFlush(flushCounter.getValue());
```

Comment:
the `waitForJournalFlush` method is also used in the `BlockMaster`, which currently does not use the same idiom so I kept the original signature of the method ... what do you mean by "more type-safe"?